sdks/java/io/amazon-web-services2/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
You need to add this file extension to the RAT plugin exclusions
https://github.com/apache/beam/blob/989c928f87b1e195d06862414b3ed6545e60ab2f/build.gradle#L108